site/en/blog/lighthouse-8-4/index.md

```diff
@@ -0,0 +1,54 @@
+---
+layout: 'layouts/blog-post.njk'
+title: What's new in Lighthouse 8.4
+authors:
+  - brendankenny
+date: 2021-09-09
+description: >
+  Lighthouse 8.4 now guards against lazily-loading LCP images and failing FID without a mobile viewport.
+tags:
+  - new-in-lighthouse
+  - lighthouse
+  # - chrome-95
+---
+
+Lighthouse 8.4 is available immediately on the command line (`npm install -g lighthouse`) and in Chrome Canary, expected to ship in Chrome 95, and to be available in [PageSpeed Insights](https://developers.google.com/speed/pagespeed/insights/) within two weeks.
+
+See the full list of changes in the [8.4 changelog](https://github.com/GoogleChrome/lighthouse/releases/tag/v8.4.0).
+
+## New Audits
```

Comment:
```suggestion
Lighthouse is an automated website auditing tool that helps developers with opportunities and diagnostics to improve the user experience of their sites. It's available in Chrome DevTools, npm (as a Node module and a CLI), or as a browser extension (in [Chrome](https://chrome.google.com/webstore/detail/lighthouse/blipmdconlkpinefehnmjammfjpmpbjk) and [Firefox](https://addons.mozilla.org/en-US/firefox/addon/google-lighthouse/)). It powers many Google services, including [web.dev/measure](https://web.dev/measure/) and [PageSpeed Insights](https://developers.google.com/speed/pagespeed/insights).

Lighthouse 8.4 is available immediately on the command line (`npm install -g lighthouse`) and in [Chrome Canary](https://www.google.com/chrome/canary/).
It will land in Chrome Stable in Chrome 95 be available in [PageSpeed Insights](https://developers.google.com/speed/pagespeed/insights/) within two weeks.

To try the Lighthouse Node CLI, use the following commands:

```text
npm install -g lighthouse
lighthouse https://www.example.com --view
```

See the full list of changes in the [8.4 changelog](https://github.com/GoogleChrome/lighthouse/releases/tag/v8.4.0).

## New audits
```